lib/rule-tester/rule-tester.js

```diff
@@ -179,6 +183,68 @@ function sanitize(text) {
     );
 }
 
+/**
+ * Define `start`/`end` properties as throwing error.
+ * @param {string} objName Object name used for error messages.
+ * @param {ASTNode} node The node to define.
+ * @returns {void}
+ */
+function defineStartEndAsError(objName, node) {
+    Object.defineProperty(node, "start", {
+        get() {
+            throw new Error(`Use ${objName}.range[0] instead of ${objName}.start`);
+        },
+        configurable: true,
+        enumerable: false
+    });
+    Object.defineProperty(node, "end", {
```

Comment:
nits: seems it could simply use `Object.defineProperties()`. :)

Reply:
Done, thanks! :)